app/templates/components/forms/events/view/create-access-code.hbs

```diff
@@ -60,65 +40,40 @@
     {{/each}}
   </div>
   <div>
-    <div class="inline field">
-      <UiCheckbox
-        @class="toggle"
-        @label={{t "Show more options"}}
-        @checked={{this.showMoreOptions}}
-        @onChange={{action (mut this.showMoreOptions)}} />
-    </div>
-    {{#if this.showMoreOptions}}
-      {{t 'Set the min and max quantity allowed to purchase per order (Optional)'}}
-      <div class="{{unless this.device.isMobile 'fields'}}">
-        <div class="wide field {{if this.device.isMobile 'sixteen' 'five'}}">
-          <label>{{t 'Min'}}</label>
-          <Input
-            @type="number"
-            @name="min"
-            @min={{0}}
-            @value={{this.data.minQuantity}} />
-        </div>
-        {{#if this.device.isMobile}}
-          <div class="ui hidden divider"></div>
-        {{/if}}
-        <div class="wide field {{if this.device.isMobile 'sixteen' 'five'}}">
-          <label>{{t 'Max'}}</label>
-          <Input
-            @type="number"
-            @name="max"
-            @min={{0}}
-            @value={{this.data.maxQuantity}} />
-        </div>
-      </div>
-      <div class="ui hidden divider"></div>
-      {{t 'Validity Period (Optional)'}}
-      <div class="fields">
-        <div class="wide field {{if this.device.isMobile 'sixteen' 'five'}}">
-          <label>{{t 'Valid from'}}</label>
+    <div class="field">
+      <label>{{t 'Access Valid from'}}</label>
+      <div class="{{if this.device.isMobile 'grouped'}} fields">
+        <div class="{{unless this.device.isMobile 'four wide'}} field">
           <Widgets::Forms::DatePicker
             @id="start_date"
             @value={{this.data.validFromDate}}
             @rangePosition="start" />
-          <div class="ui hidden divider"></div>
+        </div>
+        <div class="{{unless this.device.isMobile 'four wide'}} field">
           <Widgets::Forms::TimePicker
             @id="start_time"
             @value={{this.data.validFromTime}}
             @rangePosition="start" />
         </div>
-        <div class="wide field {{if this.device.isMobile 'sixteen' 'five'}}">
-          <label>{{t 'Expires on'}}</label>
+      </div>
+    </div>
+    <div class="field">
+       <label>{{t 'Access Expires on'}}</label>
```

Comment:
Pls fix the indentation and finalize the PR

Reply:
Okay thanks

Reply:
I'd just fixed it

Reply:
But how to finalize PR?